solarforecastarbiter/datamodel.py

```diff
@@ -1230,6 +1231,14 @@ def __post_init__(self):
                 'One of observation_id OR aggregate_id must be set')
 
 
+def __check_plot_spec__(plot_spec):
+    """Ensure that the provided plot specification is valid JSON"""
+    try:
+        json.loads(plot_spec)
```

Comment:
maybe https://python-jsonschema.readthedocs.io/en/stable/ instead?

Reply:
That's an interesting implementation of a 'JSON schema' parser. It doesn't seem to enforce the very opinionated specification proposed by (JSON Schema)[https://json-schema.org/] . 

From tinkering with this briefly, it looks like we'd still have to load the json string into a python dict and then call `validate(instance=json_dict, schema={"type":"object"})` to use this effectively. It does look like it'll give us extra safety though, so that slight performance hit may be worth it?